### PR TITLE
Add PVR recordings preview image

### PR DESCRIPTION
**Pull request by popy2k14**
Hey Pier.

Here is the PR for PVR recordings preview images.

Please review and commit :-)

thx
pOpY

### CODE REVIEW

**Comment by vonH** [comment]
@xhaggi  please review

**Comment by xhaggi** [comment]
it would be great to only show the thumbnail if one is available, because not each backend supports it.

**Comment by popy2k14** [comment]
Hey guys.

I have commited changes so the preview image is just shown when there is one available.
the visibility condition checks the texture filename and it must not contain "DefaultVideo" to show the preview image.
I have found no other way to handle this. It works here.
Maybe there is a better way, if so, please let me know.
Can anybody of you test this with an other backend as mediaportal.
I have mediaportal backend and it works like a charm for me.

thx
pOpY

File: 1080i/MyPVRRecordings.xml [hunk 1]
```diff
@@ -52,8 +52,36 @@
...
                             <scroll>false</scroll>
                         </control>
+                            <control type="image" id="123">
+                            <posx>30</posx>
+                            <posy>78</posy>
+                            <width>516</width>
+                            <height>230</height>
+                            <aspectratio aligny="bottom">keep</aspectratio>
+                            <fadetime>IconCrossfadeTime</fadetime>
+                            <texture fallback="DefaultVideoCover.png">$INFO[Container(50).ListItem.Icon]</texture>
```
Comment (xhaggi) [inline 1080i/MyPVRRecordings.xml:62]: remove the fallback attribute

File: 1080i/MyPVRRecordings.xml [hunk 2]
```diff
@@ -52,8 +52,36 @@
...
+                            <bordertexture>windows/common/darkgrey.png</bordertexture>
+                            <bordersize>1</bordersize>
+                            <visible>!SubString(Control.GetLabel(123),DefaultVideo)</visible>
```
Comment (xhaggi) [inline 1080i/MyPVRRecordings.xml:65]: `!IsEmpty(Container(50).ListItem.Icon)`

File: 1080i/MyPVRRecordings.xml [hunk 3]
```diff
@@ -52,8 +52,36 @@
                             <shadowcolor>22000000</shadowcolor>
                             <scroll>false</scroll>
                         </control>
+                            <control type="image" id="123">
```
Comment (xhaggi) [inline 1080i/MyPVRRecordings.xml:55]: remove the `id` attribute

File: 1080i/MyPVRRecordings.xml [hunk 4]
```diff
@@ -65,10 +93,10 @@
                             <pagecontrol>71</pagecontrol>
                             <autoscroll delay="13000" time="3000" repeat="13000">!Control.HasFocus(71)</autoscroll>
                             <font>Font-Condensed-S34</font>
-                            <visible>!IsEmpty(Container(50).ListItem.Plot)</visible>
+                            <visible>!IsEmpty(Container(50).ListItem.Plot) + SubString(Control.GetLabel(123),DefaultVideo)</visible>
                             <include>visiblehidefadeinfo</include>
                         </control>
-						<control type="label">
+                            <control type="label">
```
Comment (xhaggi) [inline 1080i/MyPVRRecordings.xml:99]: wrong indentation

File: 1080i/MyPVRRecordings.xml [hunk 5]
```diff
@@ -65,10 +93,10 @@
                             <pagecontrol>71</pagecontrol>
                             <autoscroll delay="13000" time="3000" repeat="13000">!Control.HasFocus(71)</autoscroll>
                             <font>Font-Condensed-S34</font>
-                            <visible>!IsEmpty(Container(50).ListItem.Plot)</visible>
+                            <visible>!IsEmpty(Container(50).ListItem.Plot) + SubString(Control.GetLabel(123),DefaultVideo)</visible>
```
Comment (xhaggi) [inline 1080i/MyPVRRecordings.xml:96]: `+ IsEmpty(Container(50).ListItem.Icon)`

File: 1080i/MyPVRRecordings.xml [hunk 6]
```diff
@@ -52,8 +52,36 @@
...
+                            <width>516</width>
+                            <height>275</height>
+                            <info>Container(50).ListItem.Plot</info>
+                            <align>justify</align>
+                            <textcolor>aawhite</textcolor>
+                            <shadowcolor>33000000</shadowcolor>
+                            <pagecontrol>71</pagecontrol>
+                            <autoscroll delay="13000" time="3000" repeat="13000">!Control.HasFocus(71)</autoscroll>
+                            <font>Font-Condensed-S34</font>
+                            <visible>!IsEmpty(Container(50).ListItem.Plot) + !SubString(Control.GetLabel(123),DefaultVideo)</visible>
```
Comment (xhaggi) [inline 1080i/MyPVRRecordings.xml:80]: `+ !IsEmpty(Container(50).ListItem.Icon)`

**Comment by xhaggi** [comment]
please address my comments + squash your commits into one.

File: 1080i/MyPVRRecordings.xml (hunk 3 above)
Comment (xhaggi) [inline 1080i/MyPVRRecordings.xml:55]: please correct the indentation too

**Comment by vonH** [comment]
@xhaggi Any further thoughts on this?

**Pull request closed without merging**